### DISCUSSION TITLE
Network Manager StrongSwan popup window too small

### DISCUSSION THREAD

**Discussion by Ruler2112** [strongswan/strongswan#1956] [Q&A] [answered]
I have an install of Ubuntu 22.04 with LXDE desktop environment.  In order to connect to my employer's VPN, I installed strongswan via apt & it installed version 5.9.5-2.  The network manager applet was updated to allow IPSEC VPN connections to be created.  This was no problem and I was able to connect to our network.

My issue is that when I right-click on the applet, it pops open a window that lists the known wifi networks and a collapsible VPN Connection sub-menu.  Because I live in the city, there are usually a lot of SSIDs listed, but the window does not scale it's size with how many there are.  I don't know how Ubuntu handles it's scanning, but several times I've opened the menu and only my own (connected) SSID is listed - as I click on VPN Connections to expand the menu and connect, the SSIDs update and the VPN Connections option is pushed down.  This generally results in my being disconnected as it tries to connect to a neighbor's wifi network.  Under my old install (Ubuntu 14.04), the popup window was as large as was needed and the menu options didn't jump around.  (I do not have the version numbers for that installation as that hard drive is dead - apologies.)  Even when opening the popup when the SSID list is populated, I have to mouse over the down arrow and wait while it scrolls down past the list, then hope it doesn't change again before I'm able to click on VPN Connections and click on the VPN to connect to / disconnect from.  While this is certainly a minor GUI issue and not anything that affects the core functionality of the software, it is highly annoying.

Is anyone else having this same type of issue?  If so, have you found any kind of solution??

**Comment by tobiasbrunner** [comment, marked as answer]
You should probably report this issue to the NetworkManager maintainers (or maybe your distribution) as the VPN plugins like strongSwan's have no control over this popup menu.

As a workaround, you could create your own shortcuts to initiate/terminate the VPN via `nmcli connection up <vpn connection name>` (use `down` to terminate it).
